cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -413,7 +413,7 @@ def test_no_template_power_control(provider, soft_assert):
     # Check the power button with checking the quadicon
     view = navigate_to(selected_template, 'AllForProvider', use_resetter=False)
     entity = view.entities.get_entity(name=selected_template.name, surf_pages=True)
-    entity.check()
+    entity.ensure_checked()
```

Comment:
added to PRT to cover this. cfme/tests/infrastructure/test_vm_power_control.py::test_no_template_power_control
cfme/tests/infrastructure/test_vm_power_control.py::test_retire_vm_with_vm_user_role